client/src/components/QuestionsAnswers/Question.jsx

```diff
@@ -31,17 +36,53 @@ const Question = ({ productId, question, productName }) => {
       ? (increaseHelpfulness(question.question_id), setHelpful(false))
       : alert('You have already marked this question as "helpful".');
   };
+  const showAnswers = () => {
+    if (!showMoreAnswers) {
+      setAnswersCount(allAnswers.length);
+      setShowMoreAnswers(true);
+    } else {
+      setAnswersCount(2);
+      setShowMoreAnswers(false);
+    }
+  };
+  const handleShow = () => { setShow(true); };
+  const handleClose = () => { setShow(false); };
 
   return (
-    <div>
-      <div>Q: { question.question_body }</div>
-      <div onClick={ handleClick }>Helpful? <a className='helpful'><u>Yes</u></a>({ count })</div>
-      <AnswerForm
-        productId={ productId }
-        productName={ productName }
-        question={ question.question_body } />
-      <div>A: <AnswersList questionId={ question.question_id } /></div>
-    </div>
+    <Container className='question-container'>
+      <Row className='question-details'>
+        <Col md="auto" className='question-text body-color'><h3>Q:</h3></Col>
+        <Col className='question-body body-color'><h3>{ question.question_body }</h3></Col>
+        <Col md="auto" className='question-helpful clickable-color'>Helpful? <a className='helpful-text' onClick={ handleClick }><u>Yes</u> </a>({ count })</Col>
```

Comment:
This is line is doing some pretty heavy lifting. Is there any way to break it up a little?

Reply:
broke up the `<Col>` tags into separate lines.